website/content/docs/api-gateway/upgrade-specific-versions.mdx

```diff
@@ -0,0 +1,280 @@
+---
+layout: docs
+page_title: Upgrading Specific Versions
+description: >-
+  Consul api gateway versions v0.2.x introduced a change to namespace backend ref
+  enforcement that will potentially break any routes created with a cross namespace in
+  v0.1.0. This documents the flow for finding and remediating any potentially
+  broken routes
```

Comment:
It probably makes sense to generalize this since we're going to be tracking future release requirements on this page as well

Reply:
This is slightly different from the intro updated in [20f3c4c](https://github.com/hashicorp/consul/pull/12812/commits/20f3c4c105a45b0b7c8d9a79c622e2a1b4120146). I'm not even sure if this one is visible to the public or just to us

Reply:
@sarahalsmiller This is slightly different from the intro updated in [20f3c4c](https://github.com/hashicorp/consul/pull/12812/commits/20f3c4c105a45b0b7c8d9a79c622e2a1b4120146). I'm not even sure if this one is visible to the public or just to us